src/Column/ActionColumnRenderer.php

```diff
@@ -158,25 +158,48 @@ private function renderButton(ActionButton|callable $button, string $name, DataC
             return $button($url);
         }
 
-        /** @var string|Stringable $content */
-        $content = $button->content instanceof Closure
-            ? ($button->content)($context->data, $context)
-            : $button->content;
-
-        $url = $button->url instanceof Closure
-            ? ($button->url)($context->data, $context)
-            : ($button->url ?? $this->createUrl($name, $context));
-
-        $attributes = $button->attributes instanceof Closure
-            ? ($button->attributes)($context->data, $context)
-            : ($button->attributes ?? []);
+        if ($button->content instanceof Closure) {
+            /**
+             * @var string|Stringable $content We intentionally don't use stricter types for closure in `ActionButton` constructor to
+             * make it easier to use in userland.
```

Comment:
How exactly it can be easier to use?

Reply:
See https://github.com/yiisoft/yii-dataview/pull/253#discussion_r1963374437